### ISSUE TITLE
Add uv to depedencies?

### ISSUE THREAD

**Issue by Jojo144** [spiral-project/ihatemoney#1357]
Hi,
Running `make serve` on my ubuntu laptop to test last changes leads to:
```
uv run --extra dev pybabel compile -d ihatemoney/translations
make: uv: Aucun fichier ou dossier de ce nom
make: *** [Makefile:55 : build-translations] Erreur 127
```
Should we add `uv` in the dependencies? What is the recommended way to install it?

Thanks.

**Comment by almet** [comment]
Hi, and thanks for opening an issue.

This is because of the recent merge of #1329 which uses uv. On ubuntu, follow the instructions in there to install: https://docs.astral.sh/uv/getting-started/installation/#standalone-installer

**Comment by almet** [comment]
> Should we add uv in the dependencies? What is the recommended way to install it?

We should definitely find a way to document  this, as it's basically now a required step to develop, using the makefile.

**Comment by Jojo144** [comment]
But what is the recommended way to install it:
- either system wide
- or in the venv using the pip package?

**Comment by almet** [comment]
It's better to install it system wide, as it's something that will manage your virtualenvs and python environmnent. You can see it as a replacement for pip :-)

**Comment by Jojo144** [comment]
Fixed by #1358
